app/views/govuk_publishing_components/components/_accordion.html.erb

```diff
@@ -49,6 +50,24 @@
               index_section_count: items.length,
             },
           }.to_json
+
+          # Data attributes for accordion links. These have been created 
```

Comment:
I think you could get away with not having this comment, I think it's clear from the code that it's separate from the other data_attributes.

Reply:
Cool, I'll remove it